src/components/progress-flow/progress-flow-item/progress-flow-item.scss

```diff
@@ -0,0 +1,178 @@
+@use '../../../style/mixins';
+@use '../../../style/functions';
+@use '../../../style/internal/variables';
+@use '../../../style/internal/z-index';
+
+.flow-item {
+    --step-background--inactive: var(
+        --progress-flow-step-background-color--inactive,
+        rgb(var(--contrast-600))
+    );
+    --step-background--active: var(
+        --progress-flow-step-background-color--active,
+        var(--mdc-theme-primary)
+    );
+    --step-background--passed: var(
+        --progress-flow-step-background-color--passed,
+        var(--step-background--active)
+    );
+
+    --step-text--inactive: var(
+        --progress-flow-step-text-color--inactive,
+        rgb(var(--contrast-1200))
+    );
+    --step-text--active: var(
+        --progress-flow-step-text-color--active,
+        var(--mdc-theme-on-primary)
+    );
+    --step-text--passed: var(
+        --progress-flow-step-text-color--passed,
+        var(--mdc-theme-on-primary)
+    );
+
+    --step-divider-color: var(
+        --progress-flow-step-divider-color,
+        rgb(var(--contrast-100))
+    );
+
+    position: relative;
+    width: 100%;
```

Comment:
Perhaps not a merge-blocker, but this is annoying:

The component always scrolls a bit when the last step is selected 😞

https://user-images.githubusercontent.com/154725/129697868-75f9f5c8-be3c-4f0e-845f-553777de69cd.mov

Reply:
Good catch. I'll have to fix this. I might have introduced this when I fixed the last issues with Safari. 

Just to explain why it happens.

The idea is that in small screen, users usually need to see previous, current, and next steps. Since the screen is small we don't have space for everything. Therefore, the component tries to center-align the selected step.

This also helps to see the "current" status, when a step is pre-selected and user loads the page. For example in ticket-handling scenarios. Always the selected/current step will be in center.

However, sometimes (like in your screen recording) there is an invisible edge, which is added by the `divider` on the last step, that causes this.
https://user-images.githubusercontent.com/35954987/129727810-7cf31f1d-2419-42ea-9cfc-8e7bc337a55d.mov

Reply:
⚡️